.github/workflows/ci.yml

```
@@ -123,6 +125,9 @@ jobs:
           # See https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html
           UBSAN_OPTIONS: print_stacktrace=1
 
+      - name: Benchmarks
+        run: cmake --build ./build --config Release --verbose --target run_benchmark
```

```suggestion
        run: cmake --build ./build --config Release --verbose --target benchmark
```

We are executing a custom target, so we are by definition "running" it?

---

We are conflicting with `benchmark` itself haha :)

---

```cmake
target_link_libraries(sourcemeta_jsontoolkit_benchmark_jsonschema benchmark::benchmark)
```

---

Ah, crap. What about `benchmark_all`? I think that would be more CMake idiomatic